worker/worker.go

```diff
@@ -265,6 +266,7 @@ func (w *Worker) saveData(issueData issueQueryWithBefore, repository *models.Rep
 			RepositoryID: (*repository).ID,
 			ProjectID:    (*repository).ProjectID,
 			Language:     nulls.String{String: strings.ToLower(*language), Valid: *language != ""},
+                        GithubUpdatedAt: node.updatedAt, //Not sure if this is right but it matches the GraphQL documentation
```

Comment:
Almost there, the only thing missing is to fix the typo in node.updatedAt (it should be node.UpdatedAt). Then you need to convert node.UpdatedAt from a string to time ( you can check an example of this, here https://pauladamsmith.com/blog/2011/05/go_time.html ) and then assign it to GithubUpdatedAt.